src/deploy/Network.ts

```diff
@@ -99,14 +100,14 @@ export async function deployNetworkComet(
     targetReserves,
     assetConfigs,
   } = {
-    governor: timelock.address,
-    pauseGuardian: timelock.address,
+    governor: timelock ? timelock.address : ethers.constants.AddressZero,
+    pauseGuardian: timelock ? timelock.address : ethers.constants.AddressZero,
```

Comment:
Thoughts on these changes? `timelock` will be undefined if the script does not deploy a `SimpleTimelock` AND there is no existing `timelock` in `aliases`. This happens for deployments that haven't been deployed on-chain yet and already have an existing `Timelock` contract on that chain (e.g. mainnet).

Reply:
But wouldn't it be good to err, since shouldn't any deployment actually have a `timelock`?

Reply:
It wouldn't necessarily be an error, since the timelock could still be set in the configuration overrides and/or directly in `configuration.json`. Pre-launch mainnet, for example, would have no `SimpleTimelock` and no existing `timelock` in `aliases`. But we specify `governor` and `pauseGuardian` directly in the `configuration.json`.

Tbh, this is all very confusing though because there's so many ways a single value can be set, and each way having a different precedence over the other.

Reply:
It'll be an error when we try to use `timelock` and there's no address, right? Agree its confusing and I'm not sure I totally know what we are talking about or why we need so many paths, so maybe we can parking lot today.

Reply:
Yeah it'll throw an error trying to access address on an `undefined` value.